shell/utils/version.js

```diff
@@ -82,7 +83,7 @@ export function isDevBuild(version) {
 }
 
 export function getVersionInfo(store) {
-  const setting = store.getters['management/byId'](MANAGEMENT.SETTING, 'server-version');
+  const setting = store.getters['management/byId'](MANAGEMENT.SETTING, SETTING.SERVER_VERSION);
```

Comment:
e2e tests are failing - I don't think there is a `SERVER_VERSION`enum - I think you want `VERSION_RANCHER`

Reply:
Yep, definitely the wrong value used here. Thanks for the tip 🙂